public/docs/dart/latest/guide/template-syntax.jade

```diff
@@ -366,11 +366,11 @@ table
 :marked
   ## Template expression operators
   The template expression language employs a subset of Dart syntax supplemented with a few special operators
-  for specific scenarios. We'll cover two of these operators: _pipe_ and _Elvis_.
+  for specific scenarios. We'll cover two of these operators: _pipe_ and _safe navigation operator_.
 .callout.is-helpful
   header Dart difference: ?. is a Dart operator
   :marked
-    The Elvis operator (`?.`) is part of the Dart language.
+    The safe navigation operator (`?.`) is part of the Dart language.
```

Comment:
That's not the name we've used in Dart, but I can live with it.

(For the curious: The Dart spec & language tour tend to avoid using a name for it, although the language tour does once call it _conditional member access_.)